src/test/java/roomescape/MissionStepTest.java

```diff
@@ -49,31 +60,41 @@ void reservationsResponseTest() {
                 .when().get("/reservations")
                 .then().log().all()
                 .statusCode(200)
-                .body("size()", is(3)); // 아직 생성 요청이 없으니 Controller에서 임의로 넣어준 Reservation 갯수 만큼 검증하거나 0개임을 확인하세요.
+                .body("size()", is(0));
     }
 
     @Test
     @DisplayName("/reservations 추가 요청 테스트")
     void addReservationsTest() {
         RestAssured.port = this.port;
-        Map<String, String> params = new HashMap<>();
+        Map<String, String> timeParams = new HashMap<>();
+        timeParams.put("startAt", "10:00");
```

Comment:
Dto를 사용해서 body를 만들어주는 것도 가능합니다.
```suggestion
        ReservationTimeAddRequest timeParams = new ReservationTimeAddRequest("10:00");
```

Reply:
앗 이 부분은 몰랐던 부분이네요! 감사합니다!